spec/requests/api/v1/contacts/create_spec.rb

```diff
@@ -0,0 +1,44 @@
+describe 'POST api/v1/contacts', type: :request do
+  subject { post api_v1_contacts_path, params: params, as: :json }
+
+  describe 'POST create' do
+    let(:email) { Faker::Internet.email }
+    let(:body) { Faker::Lorem.sentence }
+    let(:contact) { Contact.last }
```

Comment:
Do we need this?

Reply:
Yes, those are the params sent to the request.